src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -0,0 +1,37 @@
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotMap;
+
+public class IntakeSubsystem extends SubsystemBase {
+
+    private final TalonFX intakeMotor;
+
+    private final DigitalInput rightBeamBreak;
+    private final DigitalInput leftBeamBreak;
+
+    public IntakeSubsystem() {
+        intakeMotor = new TalonFX(0); 
+        intakeMotor.setNeutralMode(NeutralModeValue.Brake);
+        //intakeMotor.setInverted(true);
+        rightBeamBreak = new DigitalInput(0);
+        leftBeamBreak = new DigitalInput(0);
+    }
+
+    public void setMotorSpeed(double speed) {
+        intakeMotor.set(speed);
+    }
+
+    public boolean getRightBeamBreak() {
```

Comment:
What does the right beam break _mean_ when it's true or false? Change the names of these functions to whatever that meaning is.

For example, a method named `isRingPresentRightSide` is pretty obvious what it means, and you can guess that `true` means you see a ring on the right side of the intake.

Exposing hardware switches directly makes the user have to know whether `true` means the beam is broken, or is not broken.

Reply:
I agree, it would be better to name method to reflect more of what the value means rather than what the value actually is. This helps users not have to look through the implementation to understand how to interface with your code.